packages/jest-diff/src/diffStrings.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const chalk = require('chalk');
+const jsDiff = require('diff');
+
+const ANNOTATION = `${chalk.red('- expected')} ${chalk.green('+ actual')}\n\n`;
+
+// diff characters if oneliner and diff lines if multiline
+function diffStrings(a: string, b: string): ?string {
+  const multiline = a.match(/[\r\n]/) !== -1 && b.indexOf('\n') !== -1;
+  let isDifferent = false;
+  let result;
+
+  if (multiline) {
```

Comment:
can you retain the comment here that we have in JasmineFormatter about why multiline does diffLines?